### PR TITLE
chore(deps): bump deps to fix several security issues

### PR DESCRIPTION
**Pull request by TennyZhuang**
Signed-off-by: TennyZhuang <zty0826@gmail.com>

I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

Fixes:

- https://github.com/risingwavelabs/risingwave/security/dependabot/22
- https://github.com/risingwavelabs/risingwave/security/dependabot/23

### CODE REVIEW

**Comment by wangrunji0408** [comment]
Seems I don't have permission to see the security alerts.

**Comment by TennyZhuang** [comment]
> Seems I don't have permission to see the security alerts.

I'll take a look

File: src/workspace-hack/Cargo.toml [hunk 1]
```diff
@@ -42,7 +44,8 @@ futures-io = { version = "0.3", features = ["std"] }
 futures-sink = { version = "0.3", features = ["alloc", "std"] }
 futures-task = { version = "0.3", default-features = false, features = ["alloc", "std"] }
 futures-util = { version = "0.3", features = ["alloc", "async-await", "async-await-macro", "channel", "futures-channel", "futures-io", "futures-macro", "futures-sink", "io", "memchr", "sink", "slab", "std"] }
-hashbrown = { version = "0.12", features = ["ahash", "inline-more", "nightly", "raw"] }
+hashbrown-594e8ee84c453af0 = { package = "hashbrown", version = "0.13", features = ["ahash", "inline-more"] }
+hashbrown-5ef9efb8ec2df382 = { package = "hashbrown", version = "0.12", features = ["ahash", "inline-more", "nightly", "raw"] }
```
Comment (BugenZhao) [inline src/workspace-hack/Cargo.toml:48]: Is it possible to unify them?

Comment (wangrunji0408) [reply src/workspace-hack/Cargo.toml:48]: Looks like both versions have many crates depending on them. It is not likely to be unified.

Comment (TennyZhuang) [reply src/workspace-hack/Cargo.toml:48]: I've created multiple PRs but was pessimistic about that.

**Comment by TennyZhuang** [comment]
> Seems I don't have permission to see the security alerts.

I can't find a switch.

**Review by wangrunji0408 — APPROVED** [review]

**Comment by TennyZhuang** [comment]
https://github.com/guppy-rs/guppy/issues/65

😂😂😂

**Pull request merged**